src/.pre-commit-config.yaml.jinja

```diff
@@ -39,17 +39,17 @@
 {%- else %}
   {%- set repo_rev.autoflake = "v1.6.1" %}
   {%- set repo_rev.black = "22.8.0" %}
-  {%- set repo_rev.eslint = "v8.24.0" %}
+  {%- set repo_rev.eslint = "v9.11.1" %}
   {%- set repo_rev.flake8 = "3.9.2" %}
   {%- set repo_rev.flake8_bugbear = "21.9.2" %}
   {%- set repo_rev.isort = "5.12.0" %}
-  {%- set repo_rev.maintainer_tools = "d5fab7ee87fceee858a3d01048c78a548974d935" %}
+  {%- set repo_rev.maintainer_tools = "bf9ecb9938b6a5deca0ff3d870fbd3f33341fded" %}
   {%- set repo_rev.nodejs = "16.17.0" %}
-  {%- set repo_rev.odoo_pre_commit_hooks = "v0.0.25" %}
-  {%- set repo_rev.pre_commit_hooks = "v4.3.0" %}
+  {%- set repo_rev.odoo_pre_commit_hooks = "v0.0.33" %}
+  {%- set repo_rev.pre_commit_hooks = "v4.6.0" %}
   {%- set repo_rev.prettier = "2.7.1" %}
   {%- set repo_rev.prettier_xml = "2.2.0" %}
-  {%- set repo_rev.pylint_odoo = "v8.0.19" %}
+  {%- set repo_rev.pylint_odoo = "v9.1.2" %}
```

Comment:
Is it safe to upgrade all linters for 16 and 17 too? What is the risk of making existing repos red due to new checks?

Reply:
> Is it safe to upgrade all linters for 16 and 17 too?

Is it safe to keep using the old ones? For how long should we keep using them?
I'm for using the new and shiny things, however I'll change it if you think it's better.

> What is the risk of making existing repos red due to new checks?

I'll try the update in a repo to see how much it should change to satisfy the new requirements.

I could have left everything < 18 crystallized as it was but maybe they should be updated too from time to time, don't they?

Reply:
> I could have left everything < 18 crystallized as it was but maybe they should be updated too from time to time, don't they?

I don't think we should update for previous branches unless necessary. Otherwise tiny changes in linters will make existing branches red, and this is really annoying.

So I would add a new section for 18, with the latest linters.

Reply:
Old branches won't be affected unless manually updated. And in that case, pre-commit will update things and maybe ask for new changes, but that can be considered as part of that same manual work. However, it's true that if there must be a mass update of non-pre-commit-related things, that can be problematic. Oftentimes you're the one who does that, @sbidoul, so as you wish.